config/database.php

```diff
@@ -108,6 +108,25 @@
 
     'migrations' => 'migrations',
 
+    /*
+    |--------------------------------------------------------------------------
+    | Seeder Values
+    |--------------------------------------------------------------------------
+    |
+    | These values can be used to dynamically configure some of the seeders.
+    | For example, we can decide for what course and how many registrations
+    | (drinker and non-drinker) the seeder should create.
+    |
+    */
+
+    'seeder' => [
```

Comment:
This should not be in database.seeders but either in a test config or in database.testSeeder (or something like that). Since this does not generally affect the seeders.

Reply:
Check latest commits